sdk/src/main/java/com/google/cloud/dataflow/sdk/util/Transport.java

```diff
@@ -148,6 +149,17 @@ private static ApiComponents apiComponentsFromUrl(String urlString) {
         .setGoogleClientRequestInitializer(options.getGoogleApiTrace());
   }
 
+  public static Clouddebugger.Builder newClouddebuggerClient(DataflowPipelineOptions options) {
+    return new Clouddebugger.Builder(getTransport(),
+        getJsonFactory(),
+        chainHttpRequestInitializer(
+            options.getGcpCredential(),
+            // Do not log 404. It clutters the output and is possible even required by the caller.
+            new RetryHttpRequestInitializer(ImmutableList.of(404))))
```

Comment:
I think the retry is probably necessary, so I assume you're suggesting I get rid of the "do not log 404" part? I'll do so tomorrow.